Test/default.cpp

```diff
@@ -0,0 +1,67 @@
+/* default.cpp verifies the Teensy is connected for NanoSAM II tests. 
+ * Usage:
+ *  run this script through Teensyduino to power the NS2 payload
+ *
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+
+/* - - - - - - Functions - - - - - - */
+
+/* - - - - - - main - - - - - - *
+ * Usage:
+ *  Main function entered on runtime
+ * 
+ * Inputs:
+ *  none
+ *  
+ * Outputs:
+ *  None
+ */
+ 
+ // I think I'm writing this as Arduino code not c++? 
+int aRegCurr = 17;
+int dRegCurr = 18;
+int dRegPG = 20; 
+
+void loop()
+{
+    while(true) // run main loop forever
+    {
+		// Check voltage regulator currents
+        aCurr = analogRead(aRegCurr);
+		dCurr = analogRead(dRegCurr);
+		dPG = analogRead(dRegPG);
+		
+		
+		// Print statements for confirmation
+		// TODO: Gotta be a better way to format these statements?
+		Serial.print('Analog Current Value [A]: ');
+		Serial.print(aCurr);
+		Serial.print('\n');
```

Comment:
Looks like there is string concatenation using the ' + ' operator (https://www.arduino.cc/en/Tutorial/BuiltInExamples/StringAdditionOperator), however it requires relatively specific syntax to avoid operator precedence errors. Declaring these on multiple lines appears to be the safest, and it is how the example files in the SerialFlash library does it.